### ISSUE TITLE
position penalties

### ISSUE THREAD

**Issue by missinglink** [pelias/parser#5]
we should consider applying position penalties, such as when the `postcode` comes directly before the `street` (which is very uncommon).

eg:

```
22024 main st, ca

(1.0) ➜ [ { postcode: '22024' },
  { street: 'main st' },
  { region: 'ca' } ]

(0.7) ➜ [ { housenumber: '22024' },
  { street: 'main st' },
  { region: 'ca' } ]
```

**Comment by missinglink** [comment]
The same thing applies for admin entries, eg:
position penalties can improve the scoring of parses for the US state of `Georgia` vs, the country of `Georgia`

**Comment by Joxit** [comment]
This issue has been fixed with 6b8d8d99b4be57aa1143b6ecec7f069893605aa1 ?

**Comment by missinglink** [comment]
Looks like it, I added a testcase and closed the issue :)
